include/eve/detail/function/simd/arm/neon/friends.hpp

```diff
@@ -79,6 +79,121 @@ namespace eve::detail
+    else  if constexpr( cat == category::int8x8   ) return vclt_s8(v, w);
+    else  if constexpr( cat == category::uint32x2 ) return vclt_u32(v, w);
+    else  if constexpr( cat == category::uint16x4 ) return vclt_u16(v, w);
+    else  if constexpr( cat == category::uint8x8  ) return vclt_u8(v, w);
+    else  if constexpr( cat == category::float32x2) return vclt_f32(v, w);
+#if defined(__aarch64__)
+    else  if constexpr( cat == category::float64x1) return vclt_f64(v, w);
+    else  if constexpr( cat == category::int64x1)   return vclt_s64(v, w);
+    else  if constexpr( cat == category::uint64x1)  return vclt_u64(v, w);
+    else  if constexpr( cat == category::float64x2) return vcltq_f64(v, w);
+    else  if constexpr( cat == category::int64x2)   return vcltq_s64(v, w);
+    else  if constexpr( cat == category::uint64x2)  return vcltq_u64(v, w);
+#else
+    else  if constexpr( sizeof(T) == 8 )            return map(lt, v, w);
+#endif
+  }
+
+  template<typename T, typename N, arm_abi ABI>
+  EVE_FORCEINLINE as_logical_t<wide<T, N>> self_greater ( wide<T, N, ABI> v
+                                                        , wide<T, N, ABI> w
+                                                        ) noexcept
+  {
+    constexpr auto cat = categorize<wide<T, N, ABI>>();
+
+          if constexpr( cat == category::int32x4  ) return vcgtq_s32(v, w);
+    else  if constexpr( cat == category::int16x8  ) return vcgtq_s16(v, w);
+    else  if constexpr( cat == category::int8x16  ) return vcgtq_s8(v, w);
+    else  if constexpr( cat == category::uint32x4 ) return vcgtq_u32(v, w);
+    else  if constexpr( cat == category::uint16x8 ) return vcgtq_u16(v, w);
+    else  if constexpr( cat == category::uint8x16 ) return vcgtq_u8(v, w);
+    else  if constexpr( cat == category::float32x4) return vcgtq_f32(v, w);
+    else  if constexpr( cat == category::int32x2  ) return vcgt_s32(v, w);
+    else  if constexpr( cat == category::int16x4  ) return vcgt_s16(v, w);
+    else  if constexpr( cat == category::int8x8   ) return vcgt_s8(v, w);
+    else  if constexpr( cat == category::uint32x2 ) return vcgt_u32(v, w);
+    else  if constexpr( cat == category::uint16x4 ) return vcgt_u16(v, w);
+    else  if constexpr( cat == category::uint8x8  ) return vcgt_u8(v, w);
+    else  if constexpr( cat == category::float32x2) return vcgt_f32(v, w);
+#if defined(__aarch64__)
+    else  if constexpr( cat == category::float64x1) return vcgt_f64(v, w);
+    else  if constexpr( cat == category::int64x1)   return vcgt_s64(v, w);
+    else  if constexpr( cat == category::uint64x1)  return vcgt_u64(v, w);
+    else  if constexpr( cat == category::float64x2) return vcgtq_f64(v, w);
+    else  if constexpr( cat == category::int64x2)   return vcgtq_s64(v, w);
+    else  if constexpr( cat == category::uint64x2)  return vcgtq_u64(v, w);
+#else
+    else  if constexpr( sizeof(T) == 8 )
+      return map([]<typename E>(E const& e, E const& f){ return as_logical_t<E>(e > f); }, v, w);
```

Comment:
This is not the same as for `<` Why?

Reply:
becaue I didnt updated <. This version is better as I don't reuse th elambda anywhere else